packages/front-end/components/Holdout/HoldoutSelect.tsx

```diff
@@ -24,7 +24,7 @@ export const HoldoutSelect = ({
 }) => {
   const { getDatasourceById } = useDefinitions();
   const { hasCommercialFeature } = useUser();
-  const { holdouts, experimentsMap } = useHoldouts(selectedProject);
+  const { holdouts, experimentsMap } = useHoldouts();
```

Comment:
When do we use this project prop for useHoldouts then? I guess we filter below so this looks fine and tracks with my testing.